### PR TITLE
[regression-test](decimalv3) Adding numerical precision exceeding the data type will  error case

### PR DESCRIPTION
**Pull request by smallhibiscus**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by smallhibiscus** [comment]
@JNSimba @CalvinKirs

**Review by JNSimba — DISMISSED** [review]
LGTM

File: regression-test/suites/datatype_p0/decimalv3/test_decimalv3_string.groovy [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+suite("test_decimalv3_string") {
+    def db = "test_decimalv3_string"
+    sql "CREATE DATABASE IF NOT EXISTS ${db}"
+    sql "use ${db}"
+    sql "drop table if exists replicationtesttable"
+    sql '''CREATE TABLE `replicationtesttable` (   `k1` varchar(144) NOT NULL COMMENT '_key__',   `k2` int(11) NOT NULL COMMENT '_key__',  v4 decimal(18,4) null, ) ENGINE=OLAP UNIQUE KEY(`k1`, `k2`) COMMENT 'OLAP' DISTRIBUTED BY HASH(`k1`) BUCKETS 32 PROPERTIES ( "replication_allocation" = "tag.location.default: 1", "is_being_synced" = "false", "storage_format" = "V2", "light_schema_change" = "true", "disable_auto_compaction" = "false", "enable_single_replica_compaction" = "false" ); '''
+    sql "insert into replicationtesttable (k1,k2,v4) select 'aaa',10,22500000900.000000000000000000000000000000"
+    sql "insert into replicationtesttable (k1,k2,v4) select 'bbb',20,'22500000900.000000000000000000000000000000'"
+
+    qt_decimalv3_string "select * from replicationtesttable"
```
Comment (yiguolei) [inline regression-test/suites/datatype_p0/decimalv3/test_decimalv3_string.groovy:27]: add order by please, or the result is not stable

Comment (smallhibiscus) [reply regression-test/suites/datatype_p0/decimalv3/test_decimalv3_string.groovy]: thx , done

**Review by yiguolei — APPROVED** [review]

**Comment by yiguolei** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

**Review by mrhhsg — APPROVED** [review]
LGTM

**Pull request merged**